libfreerdp/codec/rfx.c

```diff
@@ -1011,6 +1011,8 @@ BOOL rfx_process_message(RFX_CONTEXT* context, const BYTE* data, UINT32 length,
 	BOOL ok = TRUE;
 	UINT16 expectedDataBlockType = WBT_FRAME_BEGIN;
 
+	region16_init(&updateRegion);
```

Comment:
Wouldn't it be easier to just move the declaration of updateRegion into the for() loop and get rid of init/uninit here and in fail:?